samples/wifi/radio_test/sample_description.rst

```diff
@@ -175,6 +184,392 @@ Testing
+              wifi_radio_test rx_lna_gain 2
+              wifi_radio_test rx_bb_gain 10
+              wifi_radio_test rx_capture_length 64
+              wifi_radio_test rx_cap 0
+
+           The sample shows the following output:
+
+           .. code-block:: console
+
+              ************* RX capture data ***********
+              E84FE4
+              EC4F74
+              F34F24
+              FBCF04
+              044F18
+              0B8F64
+              108FD4
+              12805C
+              10C0E4
+              0C4154
+              0541A0
+              FD81C8
+              F4C1AC
+              ED4168
+              E880F8
+              E64070
+              E80FEC
+              EC0F78
+              F2CF28
+              FB8F04
+              038F1C
+              0B0F60
+              104FCC
+              12404C
+              1180D4
+              0D8150
+              06819C
+              FE01C4
+              F601B8
+              EE8174
+              E90108
+              E6C084
+              E78FFC
+              EB8F84
+              F24F28
+              FA8F04
+              030F14
+              0A8F58
+              0FCFC0
+              124048
+              1140CC
+              0DC140
+              06C19C
+              FF01C0
+              F6C1B8
+              EF017C
+              E90110
+              E6C08C
+              E78004
+              EB0F8C
+              F18F34
+              F9CF04
+              024F0C
+              0A0F50
+              0FCFBC
+              124038
+              11C0C8
+              0D8140
+              078194
+              FF81C4
+              F6C1B4
+              EEC178
+              E94114
+
+
+         .. note::
+
+            * This test requires a DUT to be connected with VSG using cabled setup. VSG should be continuously transmitting
+            * a tone or packets. Set signal power level between -35dBm to -45dBm to get non saturated samples. In the above
+            * example capture is taken for 64 samples. Each complex sample is of 24 bits.
+            * The capture shown above is just an example. The captured samples will vary from run to run
+
+
+         * To capture baseband samples with fixed RX gain, with the following configuration:
+
+           * Channel: 144
+           * LNA gain: 2
+           * BB gain: 10
+           * Capture length: 64
+
+           Execute the following sequence of commands:
+
+           .. code-block:: console
+
+              wifi_radio_test rx 0
+              wifi_radio_test tx 0
+              wifi_radio_test init 144
+              wifi_radio_test rx_lna_gain 2
+              wifi_radio_test rx_bb_gain 10
+              wifi_radio_test rx_capture_length 64
+              wifi_radio_test rx_cap 1
+
+           The sample shows the following output:
+
+           .. code-block:: console
+
+              ************* RX capture data ***********
+              E940F0
```

Comment:
These values are not of interest since they will vary from run to run. Can we use some form of <- snip -> notation to only show a few lines ?

Reply:
Done.